course/templates/course/course_cards.html

```diff
@@ -0,0 +1,50 @@
+{% load i18n %}
+{% load course %}
+
+{% for instance in instances %}
+{% with instance_url=instance|url %}
+	<div class="col-sm-6 col-md-4 col-lg-3">
+		<div class="frontpage-course card">
```

Comment:
Apparently there is a little bit of mixed tabs and spaces. The file starts and ends with tabs. I assume you copypasted the middle part? You can check the tabs in the file with `cat -t` (or all interesting bytes with `cat -A` on Linux). Here is what it looks now (tabs are printed with string `^I`):

![image](https://user-images.githubusercontent.com/2327687/84687006-a2f83080-af45-11ea-9442-765fe1136af0.png)

You can change indentation to one of them (I would prefer tab, but haven't asked what @Mankro prefers: tabs or spaces for html files?).

For example, to replace all  sequences of 4 spaces to tabs, you could execute command `sed -i 's/    /\t/g' course/templates/course/course_cards.html`.

Reply:
This was fixed now

Reply:
This was fixed yesterday as well